test cases/common/253 long output/meson.build

```diff
@@ -0,0 +1,5 @@
+project('long-stderr', 'c')
+
+dumper = executable('dumper', 'dumper.c')
+test('dump-test', dumper)
+test('dump-test-TAP', dumper, protocol : 'tap')
```

Comment:
Hm - nothing in this test verifies that the test log etc contains relevant output, correct?